assets/app/styles/_core.less

```diff
@@ -184,6 +188,10 @@ html, body {
       }
     }
   }
+  .first-section .flow {
```

Comment:
@liggitt That would match every `flow` div unfortunately. Each `flow` is a child of an intermediate element. I couldn't get adjacent selectors to work either.